dev/com.ibm.ws.session.cache_fat/bootstrapping.properties

```diff
@@ -0,0 +1,11 @@
+#Auto-generated properties by componenttest.common.apiservices.Bootstrap
+#Tue Sep 22 10:39:43 CDT 2020
+hostName=localhost
+junit_jar=../../ant_build/lib
+HOSTNAME.password=PASSWORD
+localhost.JavaHome=/Library/Java/JavaVirtualMachines/jdk1.8.0_172.jdk/Contents/Home/jre
+local.sharedLib=../../test_prereqs/moonstone/sharedlibraries
+local.java=../../fattest.simplicity
+libertyInstallPath=../build.image/wlp
+shared.resources=../../build.sharedResources/lib
+HOSTNAME.user=USER
```

Comment:
Is this file really used by anything?

Reply:
I guess this is a locally generated file that got added. Will remove this file